api/catalog/api/controllers/search_controller.py

```diff
@@ -374,7 +375,15 @@ def search(
             s = s.query("simple_query_string", fields=["tags.name"], query=tags)
 
     if settings.USE_RANK_FEATURES:
-        feature_boost = {"standardized_popularity": 10000}
+        feature_boost = {"standardized_popularity": DEFAULT_BOOST}
+        if (
+            "unstable__authority" in search_params.data
+            and search_params.data["unstable__authority"]
+        ):
```

Comment:
`unstable__authority` will always be present in `search_params.data` because it has a default value. Just the second half of the condition should suffice.

```suggestion
        if search_params.data["unstable__authority"]:
```